plugins/setup.cfg

```diff
@@ -8,9 +8,9 @@ classifiers =
 packages = find:
 include_package_data = True
 install_requires =
-    napari
+    napari[all]
     qtpy
 
 [options.entry_points]
 napari.manifest =
-    napari-ADS = plugins.napari_ADS:napari.yaml
\ No newline at end of file
+    napari-ADS = plugins.napari_ADS:napari.yaml

```

Comment:
The issue is due to the current `setup.cfg` and `napari.yaml`:

https://github.com/axondeepseg/axondeepseg/blob/442594ca2c560a3279e409008369d3b6abddc424/plugins/setup.cfg#L14-L16

https://github.com/axondeepseg/axondeepseg/blob/3809ed1463f65948a67a830b05e837ce6cda70cb/plugins/napari_ADS/napari.yaml#L3-L6

When we `pip install -e plugins/`, we're actually just installing `plugins/napari_ADS`. (Because `plugins` itself isn't a module (no `__init__.py`), Napari can't access `plugins.napari_ADS`.) But, if we remove the "`plugins.`" part, then Napari can load the plugin just fine. (Tested on both Windows and Linux.)

I'm not really sure how this was working with the on Linux? I feel like this should have been broken in the same way no matter the platform or installation method... :thinking: